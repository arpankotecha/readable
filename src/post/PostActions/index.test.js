import * as actions from './index'

it('creates a new addPost action', () => {
  const addPost = actions.addPost({"id": 1})
  expect(addPost).toMatchSnapshot()
})

it('creates a new updateVoteCount action', () => {
  const updateVoteCount = actions.updateVoteCount(1, 2)
  expect(updateVoteCount).toMatchSnapshot()
})

