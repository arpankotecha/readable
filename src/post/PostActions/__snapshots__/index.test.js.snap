// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a new addPost action 1`] = `
Object {
  "post": Object {
    "id": 1,
  },
  "type": "ADD_POST",
}
`;

exports[`creates a new updateVoteCount action 1`] = `
Object {
  "postId": 1,
  "type": "UPDATE_VOTE_COUNT",
  "voteCount": 2,
}
`;
